File: app/src/main/java/com/berkatfaatulohalawa1711010164/facevoting/ui/preLogin1.java
package com.berkatfaatulohalawa1711010164.facevoting.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import com.berkatfaatulohalawa1711010164.facevoting.R;

public class preLogin1 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pre_login1);
        ImageView btnMulai = (ImageView) findViewById(R.id.btn_mulai);

        btnMulai.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(preLogin1.this, Login.class);
                startActivity(intent);
            }
        });
    }
}

File: android/facevoting/app/src/main/java/com/berkatfaatulohalawa1711010164/facevoting/ui/preLogin1.kt
package com.berkatfaatulohalawa1711010164.facevoting.ui

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.berkatfaatulohalawa1711010164.facevoting.R

class preLogin1 : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pre_login1)
        findViewById<ImageView>(R.id.btn_mulai).setOnClickListener {
            startActivity(Intent(this, Login::class.java))
        }
    }
}
